assets/xml/objects/object_bigslime.xml

```diff
@@ -1,4 +1,5 @@
 ﻿<Root>
+    <!-- Assets for the Gekko and Mad Jelly minibosses -->
```

Comment:
```suggestion
    <!-- Assets for the Gekko, Minislime, and Bigslime -->
```
The Gekko assets for this are used for both `Gekko and Mad Jelly Miniboss` and `Gekko and Snapper Miniboss`, it just contains the generic Gekko Assets (Plus all assets for both minislime and bigslime. Snapper is a separate set of assets though)

Reply:
Yeah, I realized that, but I guess my original comment didn't make it clear that it was both Gekkos.